migration_helpers/migration_helpers.py

```diff
@@ -0,0 +1,410 @@
+import os
+import re
+from pathlib import Path
+
+from talon import Module, actions, app
+
+mod = Module()
+
+
+# custom function to format emoticon.csv values. quick and dirty.
+def emoticon_value_converter(value):
+    return f'"""{value}"""'
+
+
+known_csv_files = {
```

Comment:
Probably not worth changing, but I would argue these entries should be dataclasses, rather than dicts

Reply:
I was being lazy, but I agree this is cleaner. Fixed.